st/clitests/awsiam_spec.py

```diff
@@ -62,6 +70,35 @@ def user_tests():
 
     AwsIamTest('Delete User').delete_user("testUser").execute_test().command_is_successful()
 
+def policy_tests():
+    #create-policy
+    samplepolicy = os.path.join(os.path.dirname(__file__), 'policy_files', 'iam-policy.json')
+    samplepolicy_testing = "file://" + os.path.abspath(samplepolicy)
+    result = AwsIamTest('Create Policy').create_policy("iampolicy",samplepolicy_testing).execute_test()
+    result.command_response_should_have("iampolicy")
+    arn = (get_arn_from_policy_object(result.status.stdout))
+
+    #create-policy fails if policy with same name already exists
+    AwsIamTest('Create Policy').create_policy("iampolicy",samplepolicy_testing).execute_test(negative_case=True)\
+        .command_should_fail().command_error_should_have("EntityAlreadyExists")
+
+    #get-policy
+    AwsIamTest('Get Policy').get_policy(arn).execute_test().command_is_successful()
```

Comment:
compare the response of the command, policy name should be part of response

Reply:
@shalakadharap

Reply:
Fixed